pkg/networkmanager/dialogs-common.jsx

```diff
@@ -183,7 +184,7 @@ export const NetworkAction = ({ buttonText, iface, connectionSettings, type }) =
         let name;
         // Find the first free interface name
         for (let i = 0; i < 100; i++) {
-            name = type + i;
+            name = (type == 'wireguard' ? 'wg' : type) + i;
```

Comment:
This is a bit "meh", but unavoidable. It could get some abstraction, but let's keep it for now until this is more mature. Maybe Marius has a better idea about this.

Reply:
I renamed it to `wg` to avoid this. `wireguard` isn't used anywhere else except to distinguish it from other interfaces.

`wg` might be less clear than `wireguard` but I think it is still understandable.